recipe/meta.yaml

```
@@ -160,8 +164,11 @@ about:
     - LICENSE.txt
     - third-party-programs.txt
   summary: High level abstract threading library
+  description: |
+    oneTBB is a flexible C++ library that simplifies the work of adding parallelism to complex applications, even if you
+    are not a threading expert.
   dev_url: https://github.com/oneapi-src/oneTBB
-  doc_url: https://software.intel.com/en-us/oneapi-tbb-documentation
+  doc_url: https://www.intel.com/content/www/us/en/developer/tools/oneapi/onetbb.html
```

```suggestion
  doc_url: https://oneapi-src.github.io/oneTBB/
```

The github project points to this link.

---

great, thanks